tasks/process-spectrum.js

```diff
@@ -162,19 +162,18 @@ async function processComponent(componentPath) {
      * @type { import('./spectrum-css-converter').SpectrumCSSConverter}
      */
     for await (const conversion of conversions) {
-        // The default package file is index.css but index-base.css contains the base styles compatible with theme switching.
-        let sourcePath = require
-            .resolve(conversion.inPackage)
-            .replace('index.css', 'index-base.css');
-
+        // The default package file is index.css
+        const sourcePath = require.resolve(conversion.inPackage);
         let sourceCSS = '';
 
-        // try to find the index-base.css file
-        try {
-            sourceCSS = fs.readFileSync(sourcePath, 'utf-8');
-        } catch (error) {
```

Comment:
Try catch is not an optimal way to check if a file exists - swapped this for fs.existsSync